apps/site/pages/docs/handling-third-party-scripts.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Handling third-party scripts'
+sidebar_label: 'How To Guides'
+---
+
+import { Callout, Steps } from 'nextra-theme-docs'
+
+<header>
+
+# Handling Third-Party Scripts
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+  This documentation is currently under development.
+</Callout>
+
+In this guide, you'll learn how to add a third-party script into FastStore projects.
+
+Third-party scripts often refer to elements embedded in a webpage and served by a domain different from the one being accessed. These scripts can include social sharing buttons, video player embeds, ads, analytics, and other scripts that make the web more dynamic and interactive.
+
+## Before you start
+
+- To maintain your website's performance without sacrificing the functionality offered by third-party scripts, consider incorporating only those third-party scripts that truly enhance the value of your website.
+
+- Ensure that you are using `@faststore/core` version 2.1.82 or above.
+
+---
+
+## Step by step
+
+1. In the Starter project, inside the `src` folder, create a folder named `scripts`.
```

Comment:
```suggestion
1. In your FastStore project, inside the `src` folder, create a folder named `scripts`.
```

Reply:
Since we s[tart by saying FastStore projects](https://github.com/vtex/faststore/pull/2057/files#diff-108863128b760ea0bc8454678f75652e23fcd7240d741c984757680fc58fdcdbR18), we can continue this pattern.